### ISSUE TITLE
[REQUEST] Release content.js in a way that allows userscript loading

### ISSUE THREAD

**Issue by KairuByte** [V1P3R-FMG/free-map-genie#57]
**Is your feature request related to a problem? Please describe.**
Development addons are a little annoying in firefox, you need to load them every time you start the browser. There exist userscript loading plugins that would allow loading of the main javascript into the page.

**Describe the solution you'd like**
A standalone content.js released alongside the extensions, as an asset.

**Describe alternatives you've considered**
None really, this is a firefox limitation.

**Additional context**
Userscripts are a little more advanced, but plenty of users would utilize them.

**Comment by kauht** [comment]
I don't know anything about extension development but the normal extension works fine for me, It behaves like any other normal firefox extension, if you are talking about the v3 version, yeah I guess. I have only loaded it as a temporary addon as I have no clue what I am doing and is the only way I can get it to work.

**Comment by MrFusiion** [comment]
Yeah normally when I release the firefox extension I sign it. So it should work like any other extension except that its not listed in the extension store. And for the dev build I use webext to launch a browser that automatically loads the extension. Open a console in the project root folder and use the command `yarn start -- firefox dev` but you will need to install nodejs and yarn,`.
